entities/Order.java

```diff
@@ -0,0 +1,54 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+
+public class Order extends Base {
+    //Attributes
+
+    private Customer customerInfo;
+    private List<Product> items;
+    private Double totalAmount;
+
+    //Constructors
+
+    public Order() {
+    }
+
+    public Order(Long id, Instant createdDate, Instant updatedDate, Long createdBy, Long updatedBy, Status status, String customerInfo, List<Product> items, Double totalAmount, Date orderDate) {
+        super(id, createdDate, updatedDate, createdBy, updatedBy, status);
+        this.customerInfo = customerInfo;
+        this.items = items;
+        this.totalAmount = totalAmount;
+    }
+
+    //Methods
+
+    public String getCustomerInfo() {
+        return customerInfo;
+    }
+
+    public void setCustomerInfo(String customerInfo) {
+        this.customerInfo = customerInfo;
+    }
```

Comment:
Looks like you still have customer info?

Reply:
Modified to Customer